lib/chunkers/markdown_splitter.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Chunkers
+  class MarkdownSplitter < TextSplitter
```

Comment:
The idea here is that this is more of a template for the common types of content people might be parsing?

Reply:
Yeah. Also, semi-inspired by https://python.langchain.com/en/latest/_modules/langchain/text_splitter.html#MarkdownTextSplitter

Who knows if it's useful IRL, but I'm trying to make sure we can handle multiple types of splitting.